quickstarts/microsoft.web/storage-static-website/README.md

```diff
@@ -0,0 +1,17 @@
+# TODO: Create ReadMe
```

Comment:
QUESTION: Does this belong under microsoft.web or microsoft.storage?

Reply:
I would say storage since there's no Microsoft.Web in it... could also be a demo instead of a quickstart